flutter-for-react-native.md

```diff
@@ -430,7 +430,7 @@ new CustomCard(
     ...
 {% endprettify %}
 
-The constructor for the `CustomCard` class shows a Dart feature. The curly braces inside the constructor indicates that the parameters are optional when initialising. In order to make these fields required, we have two options- first one is to remove the curly braces from the constructor and the second is to add the `@required` to the constructor. The latter approach enables the developer to provide parameter names when using the `CustomCard` class in the app code as show in the (Usage section) example above. 
+The constructor for the `CustomCard` class shows a Dart feature. The curly braces inside the constructor indicates that the parameters are optional when initialising. In order to make these fields required, we have two options- first one is to remove the curly braces from the constructor and the second is to add the `@required` to the constructor. The latter approach enables the developer to provide parameter names when using the `CustomCard` class in the app code as shown in the (Usage section) example above. 
```

Comment:
I see no difference in these paragraphs and it's in the same location, but I do see some grammar errors. Since the errors were there already, it's fine.

Reply:
There is a `show` -> `shown` change towards the end of the last sentence

Reply:
Good catch!  (as show => as shown)